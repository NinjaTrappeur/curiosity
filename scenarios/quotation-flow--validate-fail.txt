reset
user signup alice a alice@example.com --accept-tos

user signup mila s mila@example.com --accept-tos
user update USER-2 Mila "I'm Mila, the Seller in the quotation-flow scenario."

as mila
forms quotation new --client xochitl --seller-entity one --seller-unit alpha --buyer-entity one --buyer-unit alpha
forms quotation validate TBPJLIUG
