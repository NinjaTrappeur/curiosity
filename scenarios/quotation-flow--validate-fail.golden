1: reset
State is now empty.
2: user signup alice a alice@example.com --accept-tos
User created: USER-1
Signup confirmation email enqueued: EMAIL-1
4: user signup mila s mila@example.com --accept-tos
User created: USER-2
Signup confirmation email enqueued: EMAIL-2
5: user update USER-2 Mila "I'm Mila, the Seller in the quotation-flow scenario."
User updated: USER-2
7: as mila
Modifying default user.
8: forms quotation new --client xochitl --seller-entity one --seller-unit alpha
Quotation form created: TBPJLIUG
9: forms quotation validate TBPJLIUG
The client username does not exist.
The selling entity does not exist.
The selling unit does not exist.
